### DISCUSSION TITLE
LAMP Setup

### DISCUSSION THREAD

**Discussion by AndyXheli** [immich-app/immich#3319] [Feature Request]
Install and set up software without docker like you do with Nextcloud. Docker is nice and easy, but for users or orgs who want more performance, it would be nice to have another option outside of docker. 

Linux
PHP
Apache or ngnix
Redis

Etc

**Comment by bo0tzz** [comment]
An official non-docker install is not planned, though there are some community-made versions you can find if you have a search through the discussions here.

Fwiw, docker does not have any meaningful impact on performance.

**Comment by uhthomas** [comment]
> for users or orgs who want more performance

I'd also like to clarify that containers usually do not incur much of a performance penalty, if any. The main contributing factors to performance will be the network, disk I/O and the database, all of which an alternative deployment would make no difference to.
